src/common/meta/src/error.rs

```diff
@@ -406,6 +406,44 @@ pub enum Error {
 
     #[snafu(display("Create logical tables invalid arguments: {}", err_msg))]
     CreateLogicalTablesInvalidArguments { err_msg: String, location: Location },
+
+    #[snafu(display("Invalid  node info key: {}", key))]
+    InvalidNodeInfoKey { key: String, location: Location },
+
+    #[snafu(display("Failed to parse number: {}", err_msg))]
+    ParseNum {
+        err_msg: String,
+        #[snafu(source)]
+        error: std::num::ParseIntError,
+        location: Location,
+    },
+
+    #[snafu(display("Invalid role: {}", role))]
+    InvalidRole { role: i32, location: Location },
+
+    #[snafu(display("Failed to parse {} from utf8", name))]
+    FromUtf8 {
+        name: String,
+        #[snafu(source)]
+        error: std::string::FromUtf8Error,
+        location: Location,
+    },
+
+    #[snafu(display("Failed to serialize to json: {}", input))]
+    SerializeToJson {
```

Comment:
There are `EncodeJson` and `DecodeJson` in this Error, I think we can reuse them to avoid duplicated definitions.

Reply:
Always reuse the error variants if possible.